### DISCUSSION TITLE
Kuscka is in the House - Testing since 2019

### DISCUSSION THREAD

**Discussion by Kuscka1** [inkstitch/inkstitch#3103] [General]
Hello Kalleen and everyone else.
I am Kuscka and use Inkscape for about everything. Back in 2019 I discovered Inkstitch and you Kaalleen. Ever since day one I followed along. I've sent a few mails on change suggestions, specially on fonts. Any way, I am here to help live test on the Beta. My goodness what a huge milestone Inkstitch reached!! Congrats! I've seen the tears, I've seen the joys.

I am a seasoned digitizer, also tutoring a bunch of embroidery programs. All done in private, I think its time I go live with my knowledge and give Inkstitch my thumbs-up and tutor what I learned over the years. 

Now the question: What is the process to enable me to use Inkstitch live on Youtube? I just want to follow the correct procedures, Youtube can be that "horrible-neighbour" trying to derail with copyrights etc.

Again gratz on a huge success!!

**Comment by kaalleen** [comment]
What a nice message and thanks for the thumbs up. It's always interesting to see who is involved into the project one way or the other.

On Youtube you have no restrictions through the program itself, you just need to be careful about the designs you are going to show. Do you want to have access to the Ink/Stitch YouTube channel or want us to link to yours?

**Reply by Kuscka1** [reply]
I mostly draw my own stuff, then digitize. Yah I know Youtube is a nasty one at times. I had my running's with them before. My channel is very small, I still need to build up but I will gladly share once I am running. How does the access to Inkstitch channel work?

Oh and I took it upon myself to show case your wonderful "lil" but powerfull program. See attachment
![before and after](https://github.com/user-attachments/assets/c4b1def1-5509-4af3-91eb-b2da2c736753)

Edit: Ps. The "Stitch" very narrow. I need to rework that one.

**Reply by kaalleen** [reply]
> How does the access to Inkstitch channel work?

You'll need to share your e-mail address and someone with access to the channel can add you in.
We have a Peertube channel as well. It's been a while since the last video has been uploaded: https://diode.zone/a/inkstitch

Nice, the embroidered Ink/Stitch logo. We have a file online as well: https://inkstitch.org/tutorials/resources/branding-guide/

> My channel is very small, I still need to build up but I will gladly share once I am running.

Sounds good. Keep us up to date :)

**Comment by Kuscka1** [comment]
My very first photo with Ink/Stitch, not perfect but I am amazed. Will retry soonest. 
![Face](https://github.com/user-attachments/assets/f3ae78ef-69df-4500-a54e-922a71abbc14)

**Reply by kaalleen** [reply]
Interesting. Did you stitch it out?

**Reply by Kuscka1** [reply]
Not yet, I first want to fix the bloopers. I lost a few dots there.
